MathLib/ODE/OdeSolverFactory.h

```diff
@@ -148,7 +145,7 @@ class ConcreteOdeSolver final
 	void init() override
 	{
 		Implementation::init(NumEquations);
-		Implementation::setFunction(&_handles);
+		Implementation::setFunction(_handles.get());
```

Comment:
Wouldn't `*_handles` be even better? Or do we need to allow nullptr? Sorry, I cannot remember this detail.